src/main/java/de/maxhenkel/audioplayer/command/AudioPlayerCommands.java

```diff
@@ -409,7 +410,7 @@ private static LiteralArgumentBuilder<CommandSourceStack> bulkApplyCommand(Liter
     }
 
     private static void processShulker(CommandContext<CommandSourceStack> context, ItemStack itemInHand, String actionItem, String itemTypeName, UUID soundID, @Nullable String name) {
-        ListTag shulkerContents = itemInHand.getTagElement(BlockItem.BLOCK_ENTITY_TAG).getList("Items", 10);
```

Comment:
`10` can be replaced with `Tag.TAG_COMPOUND`

Reply:
Done